doc/starterkit/k4SimDelphes/Readme.md

```diff
@@ -38,6 +38,8 @@ The arguments in this case are (in the order they are passed)
 - The pythia card that describes the physics process [as described above](#setup-and-prerequisites)
 - The name of the output file that will be created and that will contain the generated and simulated events in EDM4hep format
 
+Setting the environment variable `K4SIMDELPHES_BATCH_MODE` to anything other an empty string will reduce the verbosity of the program a bit.
```

Comment:
Instead of using any non-empty string as true, should we foresee to have, e.g. `OFF` or `0` to enable the progress bar again explicitly?

Reply:
We can, but there would be a lot of 'falsey' values to parse. `K4SIMDELPHES_BATCH_MODE=""` is less intuitive, but works the same.

Reply:
I wouldn't go for all falsey values in this case but just settle on one and interpret all the others as true. But that makes it only slightly more intuitive than the empty string, I agree.